h/templates/client/annotation.html

```diff
@@ -174,9 +192,12 @@
         </span>
       </span>
       <span class="share-dialog-wrapper" ng-if="vm.feature('direct_linking')">
-        <button class="small btn btn-clean"
+        <button class="small btn btn-clean annotation-action-btn"
```

Comment:
Can we get rid of the other CSS classes here and just add the minimal necessary set of formatting to `annotation-action-btn`?